linkerd/core/src/main/scala/io/buoyant/linkerd/Linker.scala

```diff
@@ -102,7 +103,8 @@ object Linker {
     namers: Option[Seq[NamerConfig]],
     routers: Seq[RouterConfig],
     telemetry: Option[Seq[TelemeterConfig]],
-    admin: Option[AdminConfig]
+    admin: Option[AdminConfig],
+    usage: Option[UsageDataTelemeterConfig]
```

Comment:
The telemetry section is a list of telemeter configs.  Having a list where there's a default item always included but that default item can be replaced if you add an item to the list that is the same kind... is pretty weird and complex.  I think having usage as a separate thing is cleaner.